scripts/format-values/main.go

```diff
@@ -0,0 +1,185 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	inputFilePath  string
+	chartVersion   string
+	outputFilePath string
+)
+
+const (
+	NEW_VERSION = "v1.8.0"
+	OPERATOR    = "operator"
+	STARROCKS   = "starrocks"
+)
+
+var _starrocksKeys = []string{"nameOverride", "initPassword", "timeZone", "datadog", "starrocksCluster",
+	"starrocksFESpec", "starrocksCnSpec", "starrocksBeSpec", "secrets", "configMaps", "feProxy"}
+
+var _operatorKeys = []string{"global", "timeZone", "nameOverride", "starrocksOperator"}
+
+func main() {
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, `
+This tool is used to upgrade or downgrade the version of values.yaml for kube-starrocks chart.
+
+If the chart version is less than v1.8.0, the values.yaml is in the following format:
+aaa: bbb
+xxx: yyy
+
+If the chart version is greater than or equal to v1.8.0, the values.yaml will be changed to the following format:
+operator:
+  aaa: bbb
+starrocks:
+  xxx: yyy
+
+If you want to upgrade the version of values.yaml, you can run the following command:
+./format-values --input values.yaml --version v1.8.0 --outputFilePath values_v1.8.0.yaml
+
+If you want to downgrade the version of values.yaml, you can run the following command:
+./format-values --input values.yaml --version v1.7.1 --outputFilePath values_v1.7.1.yaml
+
+[Options]
+`)
+		flag.PrintDefaults()
+	}
+	flag.StringVar(&inputFilePath, "input", "", "the input path of values.yaml for kube-starrocks chart")
+	flag.StringVar(&chartVersion, "version", "", "the chart version, which this tool will change the values.yaml to")
+	flag.StringVar(&outputFilePath, "output", "", "the output path of values.yaml for kube-starrocks chart")
+	flag.Parse()
+
+	if inputFilePath == "" || chartVersion == "" || outputFilePath == "" {
```

Comment:
if outputFilePath is not provided, can we just output to stdout?

Reply:
good suggestion